styles/controllers/grid/files/icons.less

```diff
@@ -14,36 +14,36 @@
 	/* -- Iconography: http://www.jankoatwarpspeed.com/post/2009/09/17/sixpack-status-icons.aspx --*/
 
 	.pkp_controllers_grid_files_response pkp_controllers_grid_files_success {
-		background: #daf0da url({$baseUrl}/lib/pkp/templates/images/structure/response_success.png);
```

Comment:
I'm ambivalent about changing `baseUrl` out for `base-url` when the variable is called `baseUrl` in PHP, Smarty-land, and JS -- but if you think it's justified, carry on.